pyrene/src/components/LabelAndValue/LabelAndValue.tsx

```diff
@@ -28,9 +28,9 @@ export interface LabelAndValueProps {
 const LabelAndValue: React.FC<LabelAndValueProps> = ({
   label = '', size = 'small', value = '', type = 'neutral',
 }: LabelAndValueProps) => (
-  <div styleName={`label-and-value label-and-value-${size}`}>
-    <div styleName="label">{label}</div>
-    <div styleName={clsx('value', { [`type-${type}`]: true })}>{value}</div>
+  <div className={clsx(styles['label-and-value'], styles[`label-and-value-${size}`])}>
+    <div className={styles.label}>{label}</div>
+    <div className={clsx(styles.value, { [styles[`type-${type}`]]: true })}>{value}</div>
```

Comment:
Please do the following when the `class` will be added anyway, means always `true`.

```
<div className={clsx(styles.value, styles[`type-${type}`])}>{value}</div>
```